requirements.txt

```diff
@@ -24,5 +24,6 @@ oauth2client>=4.1.3
 azure-mgmt-storage>=3.0.0
 azure-mgmt-monitor
 azure-mgmt-sql
+azure-mgmt-network
```

Comment:
Same here, please set minimum version (you can use the current version) for all libraries.